.github/workflows/main.yml

```diff
@@ -194,19 +194,19 @@ jobs:
             dotnet: '3.1.x'
             java: '8'
             node: '10'
-            python: '3.6'
+            python: '3.7'
```

Comment:
I'd rather we keep on testing with the **minimum supported** version of python outside of the python-specific sub-matrix.